app/assets/javascripts/discourse/app/templates/components/sidebar/footer.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar-footer-wrapper">
+  <div class="sidebar-footer-container">
+    <div class="sidebar-footer-links">
+      <LinkTo @route="about" title={{i18n "about.simple_title"}} class="sidebar-footer-link sidebar-footer-link-about">
+        {{i18n "about.simple_title"}}
+      </LinkTo>
+
+      <a href
+        {{action (route-action "showKeyboardShortcutsHelp")}}
+        title={{i18n "keyboard_shortcuts_help.title"}}
+        class="sidebar-footer-link sidebar-footer-link-keyboard-shortcuts" >
+
+        {{i18n "keyboard_shortcuts_help.short_title"}}
+      </a>
+    </div>
+
+    <div class="sidebar-footer-actions">
+      <DButton
+        @route="adminSiteSettings"
+        @title="admin.site_settings.title"
+        @icon="cog"
+        @class="sidebar-footer-actions-button sidebar-footer-actions-site-settings" />
```

Comment:
Should this be wrapped by an if condition that checks the user is an admin?

Reply:
Also is there a reason this is a button instead of a proper link? I think a link makes more sense here because it'll get all the native/browser shortcuts like <kbd>ctrl</kbd>+click and middle/wheel click which I some people may expect to work with something that works/behaves like a link and I believe a proper link would be better for accessibility.

Reply:
Yea I was struggling with this as well based on the given design. I've switched them around like you've suggested because we should be using the right element for the right purpose.